README.md

```diff
@@ -46,19 +46,28 @@ First Create Virtual Environment
 Then,
 
   ```bash
-  python bioamp_tool.py [options]
+  python chords.py [options]
   ```
 
 ### Options
 
 - `-p`, `--port` <port>: Specify the serial port to use (e.g., COM5, /dev/ttyUSB0).
-- `-b`, `--baudrate` <baudrate>: Set the baud rate for serial communication (default is 115200).
+- `-b`, `--baudrate` <baudrate>: Set the baud rate for serial communication (default is 230400).
 - `--csv`: Enable CSV logging. Data will be saved to a timestamped file.
 - `--lsl`: Enable LSL streaming. Sends data to an LSL outlet.
 - `-v`, `--verbose`: Enable verbose output with detailed statistics and error reporting.
-- `--gui`: Enable the real-time data plotting GUI.
 - `-t` : Enable the timer to run program for a set time in seconds.
 
+## Applications
+
```

Comment:
Here we should point the user to the application folder.

Reply:
I have done this thing in this new commit.